### PR TITLE
add Traditional Chinese

### LINKED ISSUE #661: Interface translation?
Whether to allow me to provide traditional Chinese translation?(Lychee 4)

### PR DESCRIPTION
**Pull request by ildyria**
fixes #661 

Thank you @Oliverlin67 for your good work, we are just short of the 3 following translations:

```php
			'MIGRATION_AVAILABLE' => 'Migration available!',
```

```php
			'PHOTO_EDIT_ROTATECWISE' => 'Rotate clockwise',
			'PHOTO_EDIT_ROTATECCWISE' => 'Rotate counter-clockwise',
```

### CODE REVIEW

**Comment by Oliverlin67** [comment]
`
'MIGRATION_AVAILABLE' => '可進行轉移!',
`

`
'PHOTO_EDIT_ROTATECWISE' => '順時針旋轉', 
 'PHOTO_EDIT_ROTATECCWISE' => '逆時針旋轉',
`

File: app/Locale/ChineseTraditional.php [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_MEDIUM_HIDPI' => 'Medium HiDPI',
+			'PHOTO_SMALL' => 'Thumb',
+			'PHOTO_SMALL_HIDPI' => 'Thumb HiDPI',
+			'PHOTO_THUMB' => 'Square thumb',
+			'PHOTO_THUMB_HIDPI' => 'Square thumb HiDPI',
+			'PHOTO_LIVE_VIDEO' => '實時照片的視頻部分',
+			'PHOTO_VIEW' => 'Lychee照片瀏覽：',
+
+			'PHOTO_EDIT_ROTATECWISE' => 'Rotate clockwise',
+			'PHOTO_EDIT_ROTATECCWISE' => 'Rotate counter-clockwise',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]: 'PHOTO_EDIT_ROTATECCWISE' => '逆時針旋轉',

**Review by Oliverlin67 — CHANGES_REQUESTED** [review]

File: app/Locale/ChineseTraditional.php [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_MEDIUM' => 'Medium',
+			'PHOTO_MEDIUM_HIDPI' => 'Medium HiDPI',
+			'PHOTO_SMALL' => 'Thumb',
+			'PHOTO_SMALL_HIDPI' => 'Thumb HiDPI',
+			'PHOTO_THUMB' => 'Square thumb',
+			'PHOTO_THUMB_HIDPI' => 'Square thumb HiDPI',
+			'PHOTO_LIVE_VIDEO' => '實時照片的視頻部分',
+			'PHOTO_VIEW' => 'Lychee照片瀏覽：',
+
+			'PHOTO_EDIT_ROTATECWISE' => 'Rotate clockwise',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]: 'PHOTO_EDIT_ROTATECWISE' => '順時針旋轉',

**Review by Oliverlin67 — CHANGES_REQUESTED** [review]

File: app/Locale/ChineseTraditional.php (hunk 2 above)
Comment (ildyria) [reply app/Locale/ChineseTraditional.php]: already fixed. :'D

File: app/Locale/ChineseTraditional.php (hunk 1 above)
Comment (ildyria) [reply app/Locale/ChineseTraditional.php]: already fixed. :'D

File: app/Locale/ChineseTraditional.php [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+			'CHANGE_LOGIN' => '修改登入訊息',
+			'CHANGE_SORTING' => '修改排序',
+			'SET_DROPBOX' => '設定Dropbox',
+			'ABOUT_LYCHEE' => '關於Lychee',
+			'DIAGNOSTICS' => '診斷',
+			'DIAGNOSTICS_GET_SIZE' => '請求空間使用',
+			'LOGS' => '查看日誌',
+			'SIGN_OUT' => '登出',
+			'UPDATE_AVAILABLE' => '可用更新！',
+			'MIGRATION_AVAILABLE' => 'Migration available!',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]: 'MIGRATION_AVAILABLE' => '可進行轉移!',

Comment (ildyria) [reply app/Locale/ChineseTraditional.php]: already fixed. :'D

File: app/Locale/ChineseTraditional.php (hunk 2 above)
Comment (Oliverlin67) [reply app/Locale/ChineseTraditional.php]: > 已經修復。：'D

WOW high efficiency! @~@

Comment (ildyria) [reply app/Locale/ChineseTraditional.php]: I get mails whenever there is something happening to this repository. :smile:

File: app/Locale/ChineseTraditional.php (hunk 3 above)
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]: 'MIGRATION_AVAILABLE' => '可以進行遷移!',

**Review by Oliverlin67 — CHANGES_REQUESTED** [review]

File: app/Locale/ChineseTraditional.php [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_IMPORT_PATH_RESERVED' => '給定的路徑是Lychee的保留路徑！',
+			'UPLOAD_IMPORT_UNREADABLE' => '無法讀取文件！',
+			'UPLOAD_IMPORT_FAILED' => '無法導入文件！',
+			'UPLOAD_IMPORT_UNSUPPORTED' => '不支持的文件類型！',
+			'UPLOAD_IMPORT_ALBUM_FAILED' => '無法創建相簿！',
+
+			'ABOUT_SUBTITLE' => 'Lychee自主託管的照片管理程序',
+			'ABOUT_DESCRIPTION' => '是一個免費的照片管理工具，可在您的伺服器或網站空間上運行。安裝僅需幾秒鐘。<br>上傳，管理和分享照片(例如從本機應用程序)。<br>Lychee提供您所需的一切，所有照片均安全存儲。',
+			'FOOTER_COPYRIGHT' => '本網站的照片均受版權所有',
+			'HOSTED_WITH_LYCHEE' => 'Hosted with Lychee',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'HOSTED_WITH_LYCHEE' => '使用Lychee託管照片',
```

File: app/Locale/ChineseTraditional.php [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_IMPORT_DELETE_ORIGINALS' => '刪除原件',
+			'UPLOAD_IMPORT_DELETE_ORIGINALS_EXPL' => '如果可能，原始文件將在導入後刪除。',
+			'UPLOAD_IMPORT_LOW_MEMORY' => '內存不足！',
+			'UPLOAD_IMPORT_LOW_MEMORY_EXPL' => '服務器上的導入過程已接近內存限制，並可能最終被提前終止。',
+			'UPLOAD_WARNING' => '警告',
+			'UPLOAD_IMPORT_NOT_A_DIRECTORY' => '給定的路徑不是可讀目錄！',
+			'UPLOAD_IMPORT_PATH_RESERVED' => '給定的路徑是Lychee的保留路徑！',
+			'UPLOAD_IMPORT_UNREADABLE' => '無法讀取文件！',
+			'UPLOAD_IMPORT_FAILED' => '無法導入文件！',
+			'UPLOAD_IMPORT_UNSUPPORTED' => '不支持的文件類型！',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_IMPORT_UNSUPPORTED' => '不支援的文件類型！',
```

Comment (ildyria) [reply app/Locale/ChineseTraditional.php:397]: I don't see the difference. :thinking:

File: app/Locale/ChineseTraditional.php [hunk 6]
```diff
@@ -0,0 +1,423 @@
...
+			'MOVE_ALBUM' => '移動相簿',
+			'FULLSCREEN_ENTER' => '全螢幕模式',
+			'FULLSCREEN_EXIT' => '結束全螢幕模式',
+
+			'DELETE_ALBUM_QUESTION' => '刪除相簿和照片',
+			'KEEP_ALBUM' => '保留相簿',
+			'DELETE_ALBUM_CONFIRMATION_1' => '確定要刪除相簿',
+			'DELETE_ALBUM_CONFIRMATION_2' => '以及相簿內包含的所有照片？此動作無法還原！',
+
+			'DELETE_ALBUMS_QUESTION' => '删刪除相簿和照片',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'DELETE_ALBUMS_QUESTION' => '刪除相簿和照片',
```

File: app/Locale/ChineseTraditional.php [hunk 7]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_SET_TITLE' => '設定標題',
+			'ALBUM_DESCRIPTION' => '描述',
+			'ALBUM_NEW_DESCRIPTION' => '輸入新的相簿描述：',
+			'ALBUM_SET_DESCRIPTION' => '編輯描述',
+			'ALBUM_ALBUM' => '相簿',
+			'ALBUM_CREATED' => '創建時間',
+			'ALBUM_IMAGES' => '圖片資訊',
+			'ALBUM_VIDEOS' => '影片',
+			'ALBUM_SUBALBUMS' => '子相簿',
+			'ALBUM_SHARING' => '共享',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_SHARING' => '分享',
```

File: app/Locale/ChineseTraditional.php [hunk 8]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_SHR_YES' => '是',
+			'ALBUM_SHR_NO' => '否',
+			'ALBUM_PUBLIC' => '公開',
+			'ALBUM_PUBLIC_EXPL' => '相簿可以被其他人查看，但要遵守以下限制。',
+			'ALBUM_FULL' => '原版',
+			'ALBUM_FULL_EXPL' => '提供全分辨率圖片。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_FULL_EXPL' => '提供完整解析度照片',
```

File: app/Locale/ChineseTraditional.php [hunk 9]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_HIDDEN' => '隱藏',
+			'ALBUM_HIDDEN_EXPL' => '只有使用外部連結才能查看。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_HIDDEN_EXPL' => '只有知道連結者才能檢視',
```

File: app/Locale/ChineseTraditional.php [hunk 10]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_DOWNLOADABLE' => '下載',
+			'ALBUM_DOWNLOADABLE_EXPL' => '訪客可己下載該相簿',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_DOWNLOADABLE_EXPL' => '訪客可以下載該相簿',
```

File: app/Locale/ChineseTraditional.php [hunk 11]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_ISO' => 'ISO感光度',
+			'PHOTO_SHARING' => '共享',
+			'PHOTO_SHR_PLUBLIC' => '公開',
+			'PHOTO_SHR_ALB' => '是 (相簿)',
+			'PHOTO_SHR_PHT' => '是 (照片)',
+			'PHOTO_SHR_NO' => '否',
+			'PHOTO_DELETE' => '刪除照片',
+			'PHOTO_KEEP' => '保留照片',
+			'PHOTO_DELETE_1' => '是否要刪除照片',
+			'PHOTO_DELETE_2' => '？此動作無法還原！',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'PHOTO_DELETE_2' => '此動作無法還原！',
```

File: app/Locale/ChineseTraditional.php [hunk 12]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_DELETE_ALL_1' => '是否要刪除除所有',
+			'PHOTO_DELETE_ALL_2' => '已選擇的照片？此動作無法還原！',
+			'PHOTOS_NEW_TITLE_1' => '批量編輯照片標題',
+			'PHOTOS_NEW_TITLE_2' => '已選的照片：',
+			'PHOTO_MAKE_PRIVATE_ALBUM' => '此照片位於公開相簿中。編輯所在相冊的隱私設定，將其設置為公開或私有。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'PHOTO_MAKE_PRIVATE_ALBUM' => '此照片位於公開相簿中。編輯所在相簿的隱私設定，將其設置為公開或私有。',
```

File: app/Locale/ChineseTraditional.php [hunk 13]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_SHOW_ALBUM' => '顯示相簿',
+			'PHOTO_PUBLIC' => '公開',
+			'PHOTO_PUBLIC_EXPL' => '他人可以查看照片，但受以下限制',
+			'PHOTO_FULL' => '原始',
+			'PHOTO_FULL_EXPL' => '完整解析度照片可用',
+			'PHOTO_HIDDEN' => '隱藏',
+			'PHOTO_HIDDEN_EXPL' => '只有具有直接鏈接的人才能查看此照片。',
+			'PHOTO_DOWNLOADABLE' => '允許下載',
+			'PHOTO_DOWNLOADABLE_EXPL' => '您畫廊的訪客可以下載這張照片。',
+			'PHOTO_SHARE_BUTTON_VISIBLE' => '分享按鈕可用',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'PHOTO_SHARE_BUTTON_VISIBLE' => '顯示分享按鈕',
```

File: app/Locale/ChineseTraditional.php [hunk 14]
```diff
@@ -0,0 +1,423 @@
...
+			'ERROR_DB_2' => '無法創建數據庫。請仔細檢查主機，用戶名和密碼，確保該擁有權在數據庫中添加和修改內容。',
+			'ERROR_CONFIG_FILE' => "無法保存設置。<b>'data/'</b> 拒絕訪問。請為其他用戶設置 <b>'data/'</b> 和 <b>'uploads/'</b> 目錄的讀寫權限。查看自述文件以獲取更多信息。",
+			'ERROR_UNKNOWN' => '發生未知問題！請再試一次，檢查您的安裝和服務器。請查看自述文件以獲取更多信息。',
+			'ERROR_LOGIN' => '無法保存登錄信息。請用另一個用戶名和密碼再試一次！',
+			'ERROR_MAP_DEACTIVATED' => '地圖功能已被設為停用。',
+			'ERROR_SEARCH_DEACTIVATED' => '搜索功能已在設為停用。',
+			'SUCCESS' => '好',
+			'RETRY' => '重試',
+
+			'SETTINGS_SUCCESS_LOGIN' => '登錄信息已更新',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php:262]: 'SETTINGS_SUCCESS_LOGIN' => '登入資訊已更新',

File: app/Locale/ChineseTraditional.php [hunk 15]
```diff
@@ -0,0 +1,423 @@
...
+			'LOGIN_CREATE' => '創建',
+
+			'PASSWORD_TITLE' => '當前密碼',
+			'USERNAME_CURRENT' => '當前用戶名',
+			'PASSWORD_CURRENT' => '當前密碼',
+			'PASSWORD_TEXT' => '用戶名和密碼將被修改為：',
+			'PASSWORD_CHANGE' => '修改登入訊息',
+
+			'EDIT_SHARING_TITLE' => '編輯共享',
+			'EDIT_SHARING_TEXT' => '相冊的共享屬性將被修改為：',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'EDIT_SHARING_TEXT' => '相簿的共享屬性將被修改為：',
```

File: app/Locale/ChineseTraditional.php [hunk 16]
```diff
@@ -0,0 +1,423 @@
...
+			'SHARE_ALBUM_TEXT' => '該相冊的共享屬性：',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'SHARE_ALBUM_TEXT' => '該相簿的共享屬性：',
```

File: app/Locale/ChineseTraditional.php [hunk 17]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_SHARING_CONFIRM' => '保存',
+
+			'SORT_ALBUM_BY_1' => '相簿排序',
+			'SORT_ALBUM_BY_2' => '在一個',
+			'SORT_ALBUM_BY_3' => '排序',
+
+			'SORT_ALBUM_SELECT_1' => '創建時間',
+			'SORT_ALBUM_SELECT_2' => '標題',
+			'SORT_ALBUM_SELECT_3' => '描述',
+			'SORT_ALBUM_SELECT_4' => '公开',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'SORT_ALBUM_SELECT_4' => '公開',
```

File: app/Locale/ChineseTraditional.php [hunk 18]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_MANAGE_NEW_PHOTOS' => '現在可以管理你的新照片了',
+			'UPLOAD_COMPLETE' => '上傳完成',
+			'UPLOAD_COMPLETE_FAILED' => '有幾個照片上傳失敗了',
+			'UPLOAD_IMPORTING' => '導入',
+			'UPLOAD_IMPORTING_URL' => '導入 URL',
+			'UPLOAD_UPLOADING' => '上傳中',
+			'UPLOAD_FINISHED' => '已完成',
+			'UPLOAD_PROCESSING' => '處理中',
+			'UPLOAD_FAILED' => '失敗',
+			'UPLOAD_FAILED_ERROR' => '上傳失敗。服務器傳回了一個錯誤！',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_FAILED_ERROR' => '上傳失敗。伺服器傳回了一個錯誤！',
```

File: app/Locale/ChineseTraditional.php [hunk 19]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_FAILED_WARNING' => '上傳失敗。服務器傳回了一個警告！',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_FAILED_WARNING' => '上傳失敗。伺服器傳回了一個警告！',
```

**Comment by ildyria** [comment]
@Oliverlin67 when you do a comment, you can actually suggest directly changes.
See below:
2020-07-20_1920x1080_16:12:09

File: app/Locale/ChineseTraditional.php [hunk 20]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_SKIPPED' => '已跳過',
+			'UPLOAD_ERROR_CONSOLE' => '請查看瀏覽器控制台獲取詳細信息。',
+			'UPLOAD_UNKNOWN' => '服務器返回未知響應。請查看瀏覽器控制台獲取詳細信息。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_UNKNOWN' => '伺服器傳回了未知響應。請查看瀏覽器控制台獲取詳細信息。',
```

File: app/Locale/ChineseTraditional.php [hunk 21]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_ERROR_UNKNOWN' => '上傳失敗。服務器回傳了一個未知錯誤！',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_ERROR_UNKNOWN' => '上傳失敗。伺服器回傳了一個未知錯誤！',
```

File: app/Locale/ChineseTraditional.php [hunk 22]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_IN_PROGRESS' => 'Lychee當前正在上傳！',
+			'UPLOAD_IMPORT_WARN_ERR' => '導入成功，但返回了的警告或錯誤。請查看日誌（設置->顯示日誌）以獲取詳細信息。',
+			'UPLOAD_IMPORT_COMPLETE' => '導入完成',
+			'UPLOAD_IMPORT_INSTR' => '輸入照片鏈接直接導入：',
+			'UPLOAD_IMPORT' => '導入',
+			'UPLOAD_IMPORT_SERVER' => '從服務器導入',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_IMPORT_SERVER' => '從伺服器導入',
```

File: app/Locale/ChineseTraditional.php [hunk 23]
```diff
@@ -0,0 +1,423 @@
...
+			'UPLOAD_IMPORT_SERVER_FOLD' => '文件夾中沒有可讀的文件。請查看日誌（設置->顯示日誌）以獲取詳細信息。',
+			'UPLOAD_IMPORT_SERVER_INSTR' => '此操作將導入位於以下目錄中的所有照片，文件夾和子文件夾。',
+			'UPLOAD_ABSOLUTE_PATH' => '絕對路徑',
+			'UPLOAD_IMPORT_SERVER_EMPT' => '無法導入空文件夾！',
+			'UPLOAD_IMPORT_DELETE_ORIGINALS' => '刪除原件',
+			'UPLOAD_IMPORT_DELETE_ORIGINALS_EXPL' => '如果可能，原始文件將在導入後刪除。',
+			'UPLOAD_IMPORT_LOW_MEMORY' => '內存不足！',
+			'UPLOAD_IMPORT_LOW_MEMORY_EXPL' => '服務器上的導入過程已接近內存限制，並可能最終被提前終止。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'UPLOAD_IMPORT_LOW_MEMORY_EXPL' => '伺服器上的導入過程已接近內存限制，並可能最終被提前終止。',
```

File: app/Locale/ChineseTraditional.php (hunk 4 above)
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]: 'HOSTED_WITH_LYCHEE' => '使用Lychee託管照片',

File: app/Locale/ChineseTraditional.php [hunk 24]
```diff
@@ -0,0 +1,423 @@
...
+
+			'DELETE_ALBUM_QUESTION' => '刪除相簿和照片',
+			'KEEP_ALBUM' => '保留相簿',
+			'DELETE_ALBUM_CONFIRMATION_1' => '確定要刪除相簿',
+			'DELETE_ALBUM_CONFIRMATION_2' => '以及相簿內包含的所有照片？此動作無法還原！',
+
+			'DELETE_ALBUMS_QUESTION' => '删刪除相簿和照片',
+			'KEEP_ALBUMS' => '保留相簿',
+			'DELETE_ALBUMS_CONFIRMATION_1' => '確定要刪除全部照片',
+			'DELETE_ALBUMS_CONFIRMATION_2' => '選取的相冊和其中的所有照片？此動作無法還原！',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'DELETE_ALBUMS_CONFIRMATION_2' => '選取的相簿和其中的所有照片？此動作無法還原！',
```

File: app/Locale/ChineseTraditional.php [hunk 25]
```diff
@@ -0,0 +1,423 @@
...
+			'PUBLIC' => '公開',
+			'NUM_PHOTOS' => '照片',
+
+			'CREATE_ALBUM' => '創建相簿',
+
+			'STAR_PHOTO' => '加入我的最愛',
+			'STAR' => '我的最愛',
+			'STAR_ALL' => '將已選的標記為收藏夾',
+			'TAGS' => '標籤',
+			'TAGS_ALL' => '大量標籤',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'TAGS_ALL' => '批量標籤',
```

File: app/Locale/ChineseTraditional.php [hunk 26]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_ALBUM' => '相簿',
+			'ALBUM_CREATED' => '創建時間',
+			'ALBUM_IMAGES' => '圖片資訊',
+			'ALBUM_VIDEOS' => '影片',
+			'ALBUM_SUBALBUMS' => '子相簿',
+			'ALBUM_SHARING' => '共享',
+			'ALBUM_SHR_YES' => '是',
+			'ALBUM_SHR_NO' => '否',
+			'ALBUM_PUBLIC' => '公開',
+			'ALBUM_PUBLIC_EXPL' => '相簿可以被其他人查看，但要遵守以下限制。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_PUBLIC_EXPL' => '相簿可以被其他人查看，但需遵守以下限制 : ',
```

File: app/Locale/ChineseTraditional.php [hunk 27]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_FULL' => '原版',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_FULL' => '原圖',
```

File: app/Locale/ChineseTraditional.php [hunk 28]
```diff
@@ -0,0 +1,423 @@
...
+			'ALBUM_FULL_EXPL' => '提供全分辨率圖片。',
+			'ALBUM_HIDDEN' => '隱藏',
+			'ALBUM_HIDDEN_EXPL' => '只有使用外部連結才能查看。',
+			'ALBUM_DOWNLOADABLE' => '下載',
+			'ALBUM_DOWNLOADABLE_EXPL' => '訪客可己下載該相簿',
+			'ALBUM_SHARE_BUTTON_VISIBLE' => '分享按鈕是可見的',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ALBUM_SHARE_BUTTON_VISIBLE' => '顯示分享按鈕',
```

File: app/Locale/ChineseTraditional.php [hunk 29]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_DELETE_2' => '？此動作無法還原！',
+			'PHOTO_DELETE_ALL_1' => '是否要刪除除所有',
+			'PHOTO_DELETE_ALL_2' => '已選擇的照片？此動作無法還原！',
+			'PHOTOS_NEW_TITLE_1' => '批量編輯照片標題',
+			'PHOTOS_NEW_TITLE_2' => '已選的照片：',
+			'PHOTO_MAKE_PRIVATE_ALBUM' => '此照片位於公開相簿中。編輯所在相冊的隱私設定，將其設置為公開或私有。',
+			'PHOTO_SHOW_ALBUM' => '顯示相簿',
+			'PHOTO_PUBLIC' => '公開',
+			'PHOTO_PUBLIC_EXPL' => '他人可以查看照片，但受以下限制',
+			'PHOTO_FULL' => '原始',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'PHOTO_FULL' => '原圖',
```

File: app/Locale/ChineseTraditional.php [hunk 30]
```diff
@@ -0,0 +1,423 @@
...
+			'PHOTO_EDIT_GLOBAL_SHARING_TEXT' => '可以使用全局Lychee設置微調這張照片的可見性。 下面顯示了它的當前可見性，僅供參考。',
+			'PHOTO_SHARING_CONFIRM' => '保存',
+
+			'LOADING' => '載入中',
+			'ERROR' => '錯誤',
+			'ERROR_TEXT' => '噢，似乎出了一些問題。請重整頁面後再試一次！',
+			'ERROR_DB_1' => '無法連接數據庫，訪問被拒絕。請仔細檢查主機，用戶名和密碼，確保允許從當前位置訪問。',
+			'ERROR_DB_2' => '無法創建數據庫。請仔細檢查主機，用戶名和密碼，確保該擁有權在數據庫中添加和修改內容。',
+			'ERROR_CONFIG_FILE' => "無法保存設置。<b>'data/'</b> 拒絕訪問。請為其他用戶設置 <b>'data/'</b> 和 <b>'uploads/'</b> 目錄的讀寫權限。查看自述文件以獲取更多信息。",
+			'ERROR_UNKNOWN' => '發生未知問題！請再試一次，檢查您的安裝和服務器。請查看自述文件以獲取更多信息。',
```
Comment (Oliverlin67) [inline app/Locale/ChineseTraditional.php]:
```suggestion
			'ERROR_UNKNOWN' => '發生未知問題！請再試一次，檢查您的安裝和伺服器。請查看自述文件以獲取更多信息。',
```

**Comment by Oliverlin67** [comment]
> @Oliverlin67 when you do a comment, you can actually suggest directly changes.
> See below:
> 2020-07-20_1920x1080_16:12:09

Sorry (@~@)

**Comment by ildyria** [comment]
> Sorry (@~@)

I already edited some of your comments to make them easier to integrate. :)

**Comment by Oliverlin67** [comment]
@ildyria I have changed all I found :D

**Comment by ildyria** [comment]
@Oliverlin67 can you double check again ?

**Comment by Oliverlin67** [comment]
@ildyria No translation errors~

**Pull request merged**